changelogs/fragments/224-vmware_host_iscsi.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - vmware_host_iscsi - new code module for a new feature for manage the iSCSI configuration of ESXi host
```

Comment:
I suggest instead: a new module for the ESXi hosts that is dedicated to the management of the iSCSI configuration

Reply:
Thank you for your suggestions.
I have fixed :)